src/core/client/admin/components/PaginatedSelect/PaginatedSelect.tsx

```diff
@@ -66,42 +94,61 @@ const PaginatedSelect: FunctionComponent<Props> = ({
               )}
             </Dropdown>
           </IntersectionProvider>
-        </ClickOutside>
-      )}
-    >
-      {({ toggleVisibility, ref, visible }) => (
-        <Button
-          className={cn(styles.button, className)}
-          variant="flat"
-          adornmentLeft
-          color="mono"
-          onClick={toggleVisibility}
-          ref={ref}
-          uppercase={false}
-        >
-          {Icon && (
-            <ButtonSvgIcon className={styles.buttonIconLeft} Icon={Icon} />
-          )}
-          <Flex alignItems="center" className={styles.wrapper}>
-            {selected}
+        )}
+      >
+        {({ toggleVisibility, ref }) => (
```

Comment:
I think you can just remove unused `toggleVisibility` here.

Reply:
✅